pages/[id]/edit.js

```diff
@@ -0,0 +1,36 @@
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import SnippetForm from "@/components/SnippetForm";
+
+export default function EditPage() {
+  const router = useRouter();
+  const { isReady } = router;
+  const { id } = router.query;
+  const { data: snippet, isLoading, error } = useSWR(`/api/snippets/${id}`);
+
+  async function editSnippet(event, snippetData) {
```

Comment:
```suggestion
  async function editSnippet(snippetData) {
```
You don't use `event` here, or?

Reply:
Hi there! You are right: We do not use the event here. However, we still need it as a placeholder, since we pass the editSnippet-function to the SnippetForm-component. The SnippetForm component, in turn, expects a onSubmit function with two arguments. The reason for this is that SnippetForm-component is also used on the createPage - where the createSnippet function (which is passed on to the SnippetForm component) needs the event parameter.